tests/validators/test_dict.py

```diff
@@ -185,15 +242,7 @@ def __len__(self):
     with pytest.raises(ValidationError) as exc_info:
         v.validate_python(BadMapping())
 
-    assert exc_info.value.errors(include_url=False) == [
-        {
-            'type': 'mapping_type',
-            'loc': (),
-            'msg': 'Input should be a valid mapping, error: Mapping items must be tuples of (key, value) pairs',
-            'input': HasRepr(IsStr(regex='.+BadMapping object at.+')),
-            'ctx': {'error': 'Mapping items must be tuples of (key, value) pairs'},
-        }
-    ]
+    assert exc_info.value.errors(include_url=False) == errors
```

Comment:
Comparing to `[error]` could make `pytest.mark.parametrize` arguments more readable. As far as I can see all cases have a single error only.